python/ray/tune/integration/mlflow.py

```diff
@@ -34,6 +34,13 @@ def __init__(
         )
 
 
+# Deprecate: Remove in 2.4
+@Deprecated(
+    message=(
+        "The MLflowTrainableMixin is deprecated. "
+        "Use `ray.air.integrations.mlflow.setup_mlflow` instead."
+    )
+)
```

Comment:
can we also log a warning if this is used (we missed this `setup_wandb` as well)?

Reply:
I was under the impression this was done with the decorator - but it seems it only adds this to the docs. I'll add it.